### PR TITLE
Updated Performance Run commands for LLM Inference

### PR DESCRIPTION
**Pull request by greg-serochi**
# What does this PR do?

This includes the specific run instructions to replicate the performance numbers posted on the Intel Gaudi Developer website

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: performance_inference.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Initial Setup
+
+1. Get access to an Intel Gaudi node and [setup](https://docs.habana.ai/en/latest/shared/Pull_Prebuilt_Containers.html#pulling-prebuilt-container) the Intel Gaudi PyTorch docker for release 1.16.0
+```bash
+ docker run -itd --name 116 --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice --net=host --ipc=host vault.habana.ai/gaudi-docker/1.16.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest
+ docker exec -it 116 bash
+```
+2. Since the published performance numbers for 1.16.0 used the 1.11.1 version of Optimum Habana, you will install the 1.11.1 version of the Optimum Habana library
+```bash
+ pip install optimum-habana==1.11.1
```
Comment (ssarkar2) [inline performance_inference.md:23]: 1.11.1 is an old version

File: performance_inference.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+|------------|--------------------|----------------------|-------------------|------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| LLaMA 2 7b   | 128                | 128                  | 1                 | 1230       | 13163 tokens/sec |`QUANT_CONFIG=./quantization_config/maxabs_quant.json   TQDM_DISABLE=1 python3  run_generation.py --model_name_or_path meta-llama/Llama-2-7b-hf --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --limit_hpu_graphs --bucket_size=128 --bucket_internal --max_new_tokens 128 --max_input_tokens 128 --bf16 --batch_size 1230  --use_flash_attention --flash_attention_recompute`                                                             |
```
Comment (ssarkar2) [inline performance_inference.md:56]: why use batchsize>1 for msmt?

File: performance_inference.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+|            | 2048               | 128                  | 1                 | 120        | 1360 tokens/sec  |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 120 --max_new_tokens 128 --max_input_tokens 2048 --limit_hpu_graphs`                                                                                                                                               |
+|            | 2048               | 2048                 | 1                 | 44         | 3143 tokens/sec  |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 44 --max_new_tokens 2048 --max_input_tokens 2048 --limit_hpu_graphs  --bucket_internal --bucket_size 128`                                                                                                          |
```
Comment (regisss) [inline performance_inference.md:67]: I get errors for all these commands when launching the measures with `QUANT_CONFIG=./quantization_config/maxabs_measure.json`. I followed the instructions given in the setup section. I'm on Gaudi2 1.16. What am I doing wrong?

Comment (ssarkar2) [reply performance_inference.md:67]: I found that we need to remove --limit_hpu_graphs when measuring... maybe we should add that fact in the readmes somewhere

Comment (greg-serochi) [reply performance_inference.md:67]: We will publish these instruction on the intel website instead.   We will follow up on why the --limit_hpu_graphs is causing an issue.

**Comment by greg-serochi** [comment]
closing the PR as we'll publish info on the Intel site instead

**Pull request closed without merging**